
RectFlt32 area
LocalizationOptions options
